src/test/java/org/usrv/http/ServerTests.java

```diff
@@ -301,6 +301,34 @@ void serverHandlesMultipleRequests() throws Exception {
         }
     }
 
+    @Test
+    @DisplayName("Server always returns a content-length header in response")
+    void testContentLengthHeader() throws Exception {
+
+        // call API that will return contents of test file
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:80"))
+                .GET()
+                .build();
+
+        HttpResponse<String> response = httpClient.send(request,
+                HttpResponse.BodyHandlers.ofString());
+
+        // Validate that Content-Length matches response body size
+        assertEquals(String.valueOf(response.body().length()), response.headers().firstValue("Content-Length").get());
```

Comment:
This needs a corresponding change as well, ie. check the number of bytes instead of the length of the string